scale/job/messages/purge_jobs.py

```diff
@@ -64,6 +68,7 @@ def __init__(self):
         self._count = 0
         self._purge_job_ids = []
         self.trigger_id = None
+        self.purge = False
```

Comment:
Do you need a purge flag for this message? Since it is purging, you know purge will always be true.